SpectatorView/Compositor/CompositorDLL/CompositorInterface.cpp

```diff
@@ -0,0 +1,329 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "stdafx.h"
+#include "CompositorInterface.h"
+
+
+CompositorInterface::CompositorInterface()
+{
+    wchar_t myDocumentsPath[1024];
+    SHGetFolderPathW(0, CSIDL_MYDOCUMENTS, 0, 0, myDocumentsPath);
+    outputPath = std::wstring(myDocumentsPath) + L"\\HologramCapture\\";
+
+    DirectoryHelper::CreateOutputDirectory(outputPath);
+#if USE_CANON_SDK
```

Comment:
If we decide to remove the cannon manager, we should do a deep scrub removing any of the associated macros. I think I'm pro removing it, because I don't know how we can test it.